Myers Difference Algorithm/README.md

```diff
@@ -0,0 +1,159 @@
+#  Myers Difference Algorithm
+
+Myers Difference Algorithm is an algorithm that finds a longest common subsequence(LCS) or shortest edit scripts(SES) of two sequences. MDA can accomplish this in O(ND) time, where N is the sum of the lengths of the two sequences. The common subsequence of two sequences is the sequence of elements that appear in the same order in both sequences. 
+
+For example, let's assume you have two arrays:
+
+```
+A = [1, 2, 3]
+B = [2, 3, 4]
+```
+
+The common subsequences of these two arrays are `[2]`, and `[2,3]`. The longest common sequence in this case is `[2,3]`.
+
+## Finding the length of the Longest Common Subsequence with Myers Algorithm on Edit Graph
+
+### Edit Graph
+
+MDA uses an **Edit Graph** to solve the LCS/SES problem. Below is a illustration depicting an edit graph:
+
+<img src='Images/EditGraph.png' height="400">
+
+The x-axis at the top of the graph represents one of the sequences, `X`. The y-axis at the left side of the graph represents the other sequence, `Y`. Hence, the two sequences in question is the following:
+
+```
+X = [A, B, C, A, B, B, A]
+Y = [C, B, A, B, A, C]
+```
+
+MDA generates the edit graph through the following steps:
```

Comment:
These steps are hard to understand. It seems worthwhile to explain what the blue line, green dots, dotted lines, and red number represents, just to give the reader more context before diving into the steps.

Reply:
Thank you for reviewing.

Yes, it is necessary. Understanding edit graph is important for understanding MDA.
I'll add more explanation 👍